src/videojs-contrib-hls.js

```diff
@@ -273,6 +273,7 @@ class HlsHandler extends Component {
     if (tech.options_ && tech.options_.playerId) {
       let _player = videojs(tech.options_.playerId);
 
+      _player.gapSkipper({});
```

Comment:
Only reason I have kept it here is because it is the only place we have a reference to the player.